lib/rails/generators/preview/component_generator.rb

```diff
@@ -4,15 +4,22 @@ module Preview
   module Generators
     class ComponentGenerator < ::Rails::Generators::NamedBase
       source_root File.expand_path("templates", __dir__)
+      class_option :preview_path, type: :string, desc: "Path for previews, required when multiple preview paths are configured", default: ViewComponent::Base.config.generate.preview_path
```

Comment:
Given `preview_path` is due for deprecation, maybe `preview_paths.first` might be a good option here.

```suggestion
      class_option :preview_path, type: :string, desc: "Path for previews, required when multiple preview paths are configured", default: ViewComponent::Base.config.generate.preview_paths.first
```

It might also be worth experimenting with using `preview_paths!.first` instead if the behavior feels right. Since this option appears to be for choosing from the existing `preview_paths`, I think it'd make sense in that context.

Reply:
@boardfish the deprecated `preview_path` option is for which previews VC looks up, not where previews are generated, right?

Reply:
I hesitated to used preview_paths.first since its behavior is dependent on how your app is configured.